components/post_view/reaction/reaction.test.tsx

```diff
@@ -4,18 +4,22 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
+import {Reaction as ReactionType} from 'mattermost-redux/types/reactions';
```

Comment:
Nit : please add empty line below ? It's not documented anywhere but that's what we usually do separate out imports from Mattermost-redux